dts/bindings/mtd/eeprom.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 Vestas Wind Systems A/S
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for EEPROM devices
+
+include: base.yaml
+
+properties:
```

Comment:
Is there a reason to not include `size` as a base property?

Reply:
The idea was that some EEPROM drivers (e.g. for SoC EEPROMs) might only be able to provide this at run-time. I can move it to `eeprom-base.yaml` if you feel strongly about it :)

Reply:
I wouldn't insist.  Perhaps it could be defined as optional in the base, then `required: true` added to the descriptions for drivers that expect it to come from a devicetree property.  Something like `ngpios` that was recently added in `gpio-controller` (optional in base, required in `arm,mps2-fpgaio-gpio` until all the other controllers are updated).

Reply:
@pabigot Good idea. I have changed it to match your suggestion.